person.rb

```diff
@@ -4,7 +4,7 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly include the `:rental` in the `attr_accessor`. This is to ensure that it has getter and a setter